src/frontend/e2e/page-objects/UploadPage.ts

```diff
@@ -0,0 +1,90 @@
+import {Locator, Page} from '@playwright/test'
+import {SampleData} from '../utils/schemas/sampleData'
+
+export class UploadPage {
+
+    readonly page: Page
+    readonly selectSampleFiles: Locator
+    readonly continueButton: Locator
+    readonly cancelButton: Locator
+    readonly removeAll: Locator
+    readonly samplePrivateId: string
+    readonly samplePublicId: string
+    readonly sampleCollectionDate: string
+    readonly sampleCollectionLocation: string
+    readonly sampleSequencingDate: string
+    readonly samplePrivateSwitch: string
+    readonly importedFileNameList: Locator
+    readonly collectionDateInvalidFormatMsg: string
+    readonly sequencingDateInvalidFormatMsg: string
+    readonly submitSamplesButton: Locator
+
+
+    constructor(page: Page){
+        this.page = page;
+        this.selectSampleFiles = page.locator("input[type='file']");
+        this.continueButton = page.locator("a[href$='/upload/2'] > button");
+        this.cancelButton = page.locator("a[href$='/data/samples'] > button");
+        this.removeAll = page.locator(".MuiButton-textPrimary > .MuiButton-label > .MuiButton-iconSizeMedium");
+        this.samplePrivateId = "//div[div[text()='${VAR}']]/following-sibling::div/descendant::input[@name='privateId']";
```

Comment:
These will make our tests very fragile to any UI changes which devs make on frequent basis. should be last resort when all other routes fail